### PR TITLE
Rephrase API settings

### PR DESCRIPTION
**Pull request by matboivin**
* Rephrasing
* Update variable names (`distinctAttribute`, `acceptNewFields`)
* Correct code formatting

### CODE REVIEW

File: references/ranking_rules.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 _Child route of the [settings route](/references/settings.md)._
 
-Ranking rules is a list of all the rules that contributes to the relevancy of your search. It is possible to add your own and to change the order of the rules based on your needs.
+Ranking rules are predefined rules applied to the search results in order to improve their relevance. They are customizable so the results meet your user's needs as close as possible. Ranking rules are applied in a default order which can be changed in the settings.
```
Comment (bidoubiwa) [inline references/ranking_rules.md:5]: predefined or built-in?
```suggestion
Ranking rules are built-in rules that **ensure relevancy in search results**. Ranking rules are applied in a default order which can be changed in the settings. You can add or remove rules and change their order of importance.
```

File: references/ranking_rules.md [hunk 2]
```diff
@@ -59,29 +59,29 @@ Update the ranking rules of an index.
 
 #### Body
 
-List of ranking rules in order of importance.
+An array that contain ranking rules sorted by order of importance.
 
-To add your own ranking rule you need to communicate the field and if its value is descending or ascending.
+To add your own ranking rule, you have to communicate either `asc` for ascending order or `dsc` for descending order followed by the field name in brackets.
```
Comment (bidoubiwa) [inline references/ranking_rules.md:64]:
```suggestion
To add your own ranking rule, you have to communicate either `asc` for ascending order or `desc` for descending order followed by the field name in brackets.
```

File: references/ranking_rules.md [hunk 3]
```diff
@@ -59,29 +59,29 @@ Update the ranking rules of an index.
...
 
-For ascending it would be: `asc(price)`
+- To apply an **ascending sorting** (results sorted by increasing value of the attribute): `asc(attribute_name)`
 
-For descending it would be: `dsc(release_date)`
+- To apply a **descending sorting** (results sorted by decreasing value of the attribute): `dsc(attribute_name)`
```
Comment (bidoubiwa) [inline references/ranking_rules.md:68]:
```suggestion
- To apply a **descending sorting** (results sorted by decreasing value of the attribute): `desc(attribute_name)`
```

File: references/ranking_rules.md [hunk 4]
```diff
@@ -59,29 +59,29 @@ Update the ranking rules of an index.
...
-            "dsc(rank)"
-        ]'
+      "typo",
+      "words",
+      "proximity",
+      "attribute",
+      "wordsPosition",
+      "exactness",
+      "asc(release_date)",
+      "dsc(rank)"
```
Comment (bidoubiwa) [inline references/ranking_rules.md:83]:
```suggestion
      "desc(rank)"
```

File: references/ranking_rules.md [hunk 5]
```diff
@@ -102,13 +102,13 @@ Reset the [ranking rules](/guides/main_concepts/relevancy.md#ranking-rules) of a
 
 #### Default value
 
-Array with the [built-in ranking rules](/guides/main_concepts/relevancy.md#order-of-the-rules) ordered by importance.
+An array that contain [built-in ranking rules](/guides/main_concepts/relevancy.md#order-of-the-rules) sorted by order of importance.
```
Comment (bidoubiwa) [inline references/ranking_rules.md:105]: Not sure
```suggestion
An array that contains [built-in ranking rules](/guides/main_concepts/relevancy.md#order-of-the-rules) sorted by order of importance.
```

File: references/settings.md [hunk 6]
```diff
@@ -84,15 +84,15 @@ Update the settings of an index.
...
-| **indexNewFields**       | Boolean   | New fields in newly added document are/aren't added to MeiliSearch           | `true`                                                                                        |
+| Variable                 | Type      | Description                                                                      | Default value                                                                                     |
+| ------------------------ | --------- | -------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------- |
+| **synonyms**             | Object    | List of associated words that should be treated similarly in a search query      | `{}`                                                                                              |
+| **stopWords**            | [Strings] | List of words that should be excluded from search queries                        | `[]`                                                                                              |
```
Comment (bidoubiwa) [inline references/settings.md:90]:
```suggestion
| **stopWords**            | [Strings] | List of words ignored by MeiliSearch when present in search queries                        | `[]`                                                                                              |
```

File: references/settings.md [hunk 7]
```diff
@@ -162,15 +162,15 @@ Reset the settings of an index.
...
-| **indexNewFields**       | New fields in newly added document are/aren't added to MeiliSearch           | `true`                                                                                        |
+| Variable                 | Description                                                                      | Default value                                                                                     |
+| ------------------------ | -------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------- |
+| **synonyms**             | List of associated words that should be treated similarly in a search query      | `{}`                                                                                              |
+| **stopWords**            | List of words that should be excluded from search queries ignored                | `[]`                                                                                              |
```
Comment (bidoubiwa) [inline references/settings.md:168]:
```suggestion
| **stopWords**            | List of words ignored by MeiliSearch when present in search queries                | `[]`                                                                                              |
```

File: references/settings.md [hunk 8]
```diff
@@ -162,15 +162,15 @@ Reset the settings of an index.
...
-| **synonyms**             | List of associated words that are considered the same in a search query      | `{}`                                                                                          |
-| **indexNewFields**       | New fields in newly added document are/aren't added to MeiliSearch           | `true`                                                                                        |
+| Variable                 | Description                                                                      | Default value                                                                                     |
+| ------------------------ | -------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------- |
+| **synonyms**             | List of associated words that should be treated similarly in a search query      | `{}`                                                                                              |
```
Comment (bidoubiwa) [inline references/settings.md:167]:
```suggestion
| **synonyms**             | List of associated words treated similarly  | `{}`                                                                                              |
```

File: references/settings.md [hunk 9]
```diff
@@ -162,15 +162,15 @@ Reset the settings of an index.
...
+| **stopWords**            | List of words that should be excluded from search queries ignored                | `[]`                                                                                              |
+| **rankingRules**         | List of ranking rules sorted by order of importance                              | [A list of ordered built-in ranking rules](/guides/main_concepts/relevancy.md#order-of-the-rules) |
+| **distinctAttribute**    | Returns only distinct (different) values of a given field                        | `null`                                                                                            |
```
Comment (bidoubiwa) [inline references/settings.md:170]:
```suggestion
| **distinctAttribute**    | Search returns documents with distinct (different) values of a given field                        | `null`                                                                                            |
```

File: references/settings.md [hunk 10]
```diff
@@ -84,15 +84,15 @@ Update the settings of an index.
...
+| ------------------------ | --------- | -------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------- |
+| **synonyms**             | Object    | List of associated words that should be treated similarly in a search query      | `{}`                                                                                              |
+| **stopWords**            | [Strings] | List of words that should be excluded from search queries                        | `[]`                                                                                              |
+| **rankingRules**         | [Strings] | List of ranking rules sorted by order of importance                              | [A list of ordered built-in ranking rules](/guides/main_concepts/relevancy.md#order-of-the-rules) |
+| **distinctAttribute**    | String    | Returns only distinct (different) values of the given field                      | `null`                                                                                            |
```
Comment (bidoubiwa) [inline references/settings.md:92]:
```suggestion
| **distinctAttribute**    | String    | Search returns documents with distinct (different) values of the given field                      | `null`                                                                                            |
```

File: references/settings.md [hunk 11]
```diff
@@ -84,15 +84,15 @@ Update the settings of an index.
...
-| **synonyms**             | Object    | List of associated words that are considered the same in a search query      | `{}`                                                                                          |
-| **indexNewFields**       | Boolean   | New fields in newly added document are/aren't added to MeiliSearch           | `true`                                                                                        |
+| Variable                 | Type      | Description                                                                      | Default value                                                                                     |
+| ------------------------ | --------- | -------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------- |
+| **synonyms**             | Object    | List of associated words that should be treated similarly in a search query      | `{}`                                                                                              |
```
Comment (bidoubiwa) [inline references/settings.md:89]:
```suggestion
| **synonyms**             | Object    | List of associated words treated similarly      | `{}`                                                                                              |
```

File: references/ranking_rules.md (hunk 1 above)
Comment (matboivin) [reply references/ranking_rules.md:5]: I will use built-in everywhere

**Pull request merged**